### DISCUSSION TITLE
Payload exit without message when using sendEmail in unconfigured server

### DISCUSSION THREAD

**Discussion by thgh** [payloadcms/payload#3788] [General]
### Link to reproduction

todo

### To Reproduce

* Don't configure email using `payload.init({ email })`
* Call `payload.sendEmail()`

### Describe the Bug

Expected: sendEmail throws Error

Actual: payload exits "successfully"

### Payload Version

1.15.5

**Comment by jmikrut** [comment]
Hey @thgh — makes sense. We will replicate and address accordingly. Thank you!

**Comment by denolfe** [comment]
As of right now, if email is not configured, Payload will use ethereal.email (mock email) by default. This is expected behavior as of right now.

We're open to ideas on how to make this more evident to the user in the future. Transferring to a Discussion.
